src/renderer/features/multisig-wallet-pairing/components/components/Signatory.tsx

```diff
@@ -143,20 +150,37 @@ export const Signatory = ({
   useEffect(() => {
     if (isOwnAccount || contacts.length === 0 || !chain) return;
 
-    const contactsOptions: ComboboxOption[] = [];
+    const addressOptions: ComboboxItem[] = [];
     for (const contact of filteredContacts) {
       const displayAddress = toAddress(contact.accountId, { prefix: chain.addressPrefix });
       const isValidAddress = validateAddress(displayAddress, chain);
 
       if (!isValidAddress) continue;
 
-      contactsOptions.push({
+      addressOptions.push({
         id: contact.id.toString(),
-        element: <Address showIcon title={contact.name} address={displayAddress} />,
+        label: <Address showIcon title={contact.name} address={displayAddress} />,
         value: { address: displayAddress },
       });
     }
 
+    if (validateAddress(query, chain)) {
+      const displayAddress = toAddress(query, { prefix: chain.addressPrefix });
```

Comment:
```suggestion
      const displayedAddress = toAddress(query, { prefix: chain.addressPrefix });
```

otherwise sounds like a boolean condition

Reply:
This is why I prefer to name boolean values like isSomething, hasSomething, shouldShomething.
Would be nice to have this rule in spektr codebase